.github/workflows/on_pull_request.yml

```diff
@@ -285,13 +285,18 @@ jobs:
             */*/node_modules
           key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
 
+      - name: Get playwright version
+        id: pw-version
+        run: |
+          echo "::set-output name=version::$(yarn playwright --version)"
+        shell: bash
+
       - name: Cache playwright binaries
         uses: actions/cache@v2
         id: playwright-cache
         with:
-          path: |
-            ~/.cache/ms-playwright
-          key: cache-playwright-linux-1.19.2
+          path: "~/.cache/ms-playwright"
+          key: cache-playwright-${{ runner.os }}-${{ steps.pw-version.outputs.version }}
```

Comment:
Since this is caching playwright we are only interested in invalidating it when that dep changes. That's why the installed version number is used as part of the key and not the whole lockfile.